opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/trigger-test.js

```diff
@@ -118,11 +119,54 @@ describe('Trigger resolver standard behavior', () => {
     expect(queryResult.data.trigger).not.toBeNull();
     expect(queryResult.data.trigger.id).toEqual(triggerInternalId);
   });
-  it('should list triggers', async () => {
-    const queryResult = await queryAsAdmin({ query: LIST_QUERY, variables: { first: 10 } });
-    expect(queryResult.data.triggers.edges.length).toEqual(2);
+  // Group trigger start
+  it('security user should create group trigger', async () => {
+    // Create the trigger
+    const GROUP_TRIGGER_TO_CREATE = {
+      input: {
+        name: 'group trigger',
+        description: '',
+        event_types: 'create',
```

Comment:
EVENT_TYPE_CREATE instead of 'create' to use the constant

Reply:
fixed